common/extensions/api/_ipfs_api_features.json

```diff
@@ -54,5 +54,38 @@
     "contexts": ["blessed_extension"],
     "dependencies": [],
     "allowlist": ["780BF954C0F7C586EA9662D4F967771F49CC2114", "FF32507DC3DB5DFFD1D6733187C84D4B74713D63", "A321D47A2B4CA86898167A55CA8B2E02385EA7CD"]
+  }],
+  "ipfs.getIpnsKeysList": [{
+    "channel": "stable",
+    "contexts": ["blessed_extension"],
+    "dependencies": ["permission:ipfs"]
```

Comment:
I don't think we want to expose anything new unless we have a security review for it and need to for the chrome.ipfs API